samples/drivers/jesd216/boards/nrf52840dk_nrf52840_spi.overlay

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* This DTS overlay allows to test MX25R6435F  using SPI_NOR driver */
+
+&pinctrl {
+	spi1_alt_default: spi1_alt_default {
+		group1 {
+			psels = <NRF_PSEL(SPIM_SCK, 0, 19)>,
+				<NRF_PSEL(SPIM_MOSI, 0, 20)>,
+				<NRF_PSEL(SPIM_MISO, 0, 21)>;
+		};
+	};
+
+	spi1_alt_sleep: spi1_alt_sleep {
+		group1 {
+			psels = <NRF_PSEL(SPIM_SCK, 0, 19)>,
+				<NRF_PSEL(SPIM_MOSI, 0, 20)>,
+				<NRF_PSEL(SPIM_MISO, 0, 21)>;
+			low-power-enable;
+		};
+	};
+};
+
+/delete-node/ &mx25r64;
+
+&spi1 {
+	compatible = "nordic,nrf-spi";
+	status = "okay";
+	cs-gpios = <&gpio0 17 GPIO_ACTIVE_LOW>;
+	pinctrl-0 = <&spi1_alt_default>;
+	pinctrl-1 = <&spi1_alt_sleep>;
+	pinctrl-names = "default", "sleep";
+	mx25r64: mx25r6435f@0  {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <80000000>;
```

Comment:
```suggestion
		spi-max-frequency = <33000000>;
```
This flash chip can handle 80 MHz only in the High Performance Mode and currently the spi_nor driver is not capable of enabling this mode. In the default Ultra Low Power Mode the maximum frequency is 33 MHz.